stance/predict/overlay.py

```diff
@@ -12,8 +14,59 @@ def display_overlay(input_video, template_skeletons):
     input_video : String
         path to the video of the user performing the motion
     template_skeletons : List[Skeletons]
+    benchmark_indices : frames where benchmark skeletons should be overlayed
```

Comment:
You need the type as well. It should be like 

variable : Type
         description